public/index.html

```diff
@@ -37,8 +37,6 @@
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
 
-    <link rel="preconnect" href="https://www.google-analytics.com/" />
```

Comment:
We should probably preconnect to our existing analytics servers. Can you ticket that? No need to change anything in _this_ PR.